test/integration/testConnection.js

```diff
@@ -46,34 +46,34 @@ describe('Connection test', function ()
     var connection = snowflake.createConnection(connOption.valid);
 
     async.series([
-        function (callback)
-        {
-          connection.connect(function (err)
-          {
-            assert.ok(!err, JSON.stringify(err));
-            callback();
-          });
-        },
-        function (callback)
+      function (callback)
```

Comment:
if you change this code then let's apply eslint formatting rules

Reply:
ok